images/cilium/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:1.1-experimental
+
+# Copyright 2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG CILIUM_BUILDER_IMAGE=docker.io/cilium/cilium-builder-dev:5c1d8916b5ad0fbb86aabcc6239e163b6c645a71
+ARG CILIUM_RUNTIME_IMAGE=docker.io/cilium/cilium-runtime-dev:bf5a13ab0dd341b446ec1f40a809d365024eeb09
+
+FROM --platform=linux/amd64 ${CILIUM_BUILDER_IMAGE} as builder
+
+ARG NOSTRIP
+ARG LOCKDEBUG
+
+RUN --mount=type=bind,readwrite,target=/go/src/github.com/cilium/cilium --mount=target=/root/.cache,type=cache --mount=target=/go/pkg/mod,type=cache \
+  make clean-container build-container install-container \
+    NOSTRIP=$NOSTRIP LOCKDEBUG=$LOCKDEBUG PKG_BUILD=1 SKIP_DOCS=true DESTDIR=/out/linux/amd64
+
+RUN --mount=type=bind,readwrite,target=/go/src/github.com/cilium/cilium --mount=target=/root/.cache,type=cache --mount=target=/go/pkg/mod,type=cache \
+  env GOARCH=arm64 CC=aarch64-linux-gnu-gcc \
+    make clean-container build-container install-container \
```

Comment:
Turns out there is something odd going on and `/var/lib/cilium` is not populated properly.

Reply:
This is due to exclusion of `.git` dir, waiting for https://github.com/cilium/cilium/pull/12326.

Reply:
#12326 was merged, and I've rebased this PR just now